README.md

```diff
@@ -1,2 +1,29 @@
-# index-mermaid-graph
-a tool that produces SVG diagrams using OLM graphs as the input
+## Output Mermaid graph script from Operator Lifecycle Manager indices
+
+### Required:
+make, sqlite3, go v14+, container execution environment (such as Docker, podman)
+
+### Usage
+Adjust the sqlite file pointed to in `sqlite3.sql`, this controls what index gets graphed
+
+Then:
+```bash
+make run <ARGS=operator-package-name>
+```
+For instance:
+```bash
+make run ARGS=amq-operator
```

Comment:
We are using cobra so why not pass the values as arg of the command?

Reply:
we are, but keeping it very simple, this just passes the only accepted arg and this is the way it needs to be done through make.